css/styles.css

```diff
@@ -0,0 +1,59 @@
+body {
+  background-color: #f0f0f0;
+}
+
+.heading {
+  background-color: #562b81;
+  color: white;
+  font-size: 8px;
+}
+
+.top-header {
+  font-size: medium;
+}
+
+button {
+  background-color: #562b81;
+  color: white;
+  border: none;
+  border-radius: 5px;
+  width: 80px;
+  height: 30px;
+}
+
+.save-button {
+  display: flex;
+  justify-content: right;
+}
+
+label {
+  font-size: 15px;
+  font-weight: bold;
+}
+
+img {
+  width: 100%;
+  height: 100%;
```

Comment:
Remove this. You only need to set the `width`. The `height` will automatically adjust for us